### DISCUSSION TITLE
Translation

### DISCUSSION THREAD

**Discussion by martin66789** [mriscoc/Ender3V2S1#1093] [Q&A]
Is there any way to translate the text in the "libproui_ubl.a" file? I mean the "Lower" "Calculating relative heights" texts.
In older versions these texts were still translatable, but unfortunately I can't do it here, if I copy a single letter it gives an error when building.

**Comment by mriscoc** [comment]
Some new string constants will be available in the language_en.h file for easier translation in the next version.

**Comment by martin66789** [comment]
Thanks.

I translate several things in different files and it is quite fiddly and takes quite a long time after an update to find the texts that need to be translated. If the texts were placed in the language file, it would help a lot if it can be solved.

When the translation is finished, and if possible, I would share it in the whole package so that others can use it.
